alts/src/generated/main/grpc/io/grpc/alts/internal/HandshakerServiceGrpc.java

```diff
@@ -157,6 +172,39 @@ public io.grpc.stub.StreamObserver<io.grpc.alts.internal.HandshakerReq> doHandsh
   /**
    * A stub to allow clients to do synchronous rpc calls to service HandshakerService.
    */
+  public static final class HandshakerServiceBlockingV2Stub
+      extends io.grpc.stub.AbstractBlockingStub<HandshakerServiceBlockingV2Stub> {
```

Comment:
AbstractBlockingStub just has newStub methods.  I don't think that there is any new behavior that we need to capture.  The V2 for the generated stubs was because we changed the return type of blockingServerStreamingCall.